README.md

```diff
@@ -40,7 +40,7 @@ ${ZKLLVM_BUILD:-build}/bin/assigner/assigner -b ${ZKLLVM_BUILD:-build}/examples/
 
     Install fmt-6
     ```
-    wget http://launchpadlibrarian.net/466750967/libfmt-dev_6.1.2+ds-2_amd64.deb
+    wget http://archive.ubuntu.com/ubuntu/pool/universe/f/fmtlib/libfmt-dev_6.1.2
```

Comment:
But the installation through apt install does not require it, right? I mean, apt should automatically install all the dependencies.

Reply:
Nope, apt will not find version 6, because it's too old for Ubuntu 22.04, apt will see only versions 8+. I can add an old repository before "apt update", but I think it's better to manually download this single package, than adding an old repo to sources list.

Reply:
Can't we use an older version?

Reply:
Nope, we have copied the code of seastar, we will need to update it, it's long...

Reply:
Builds and produces the proof just fine inside clean ubuntu:22.04 with fmt of version 8.1.1